Project.toml

```diff
@@ -12,9 +12,11 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-JuMP = "0.20"
-PowerModels = "v0.12.5"
-julia = "1.2"
+Ipopt = "0.5, 0.6"
+JuMP = "0.19, 0.20"
+OrderedCollections = "1.0, 1.1"
```

Comment:
```suggestion
OrderedCollections = "1.0"
```
that is the same

Reply:
Right! I always forgot this living in my `pre-1.0` release world :P